src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -0,0 +1,565 @@
+        and `labels`
+
+        Args:
+            masks (`torch.Tensor`):
+                A tensor of shape `(num_queries, height, width)`.
+            scores (`torch.Tensor`):
+                A tensor of shape `(num_queries)`.
+            labels (`torch.Tensor`):
+                A tensor of shape `(num_queries)`.
+            object_mask_threshold (`float`):
+                A number between 0 and 1 used to binarize the masks.
+
+        Raises:
+            `ValueError`: Raised when the first dimension doesn't match in all input tensors.
+
+        Returns:
+            `Tuple[`torch.Tensor`, `torch.Tensor`, `torch.Tensor`]`: The `masks`, `scores` and `labels` without the
+            region < `object_mask_threshold`.
+        """
+        if not (masks.shape[0] == scores.shape[0] == labels.shape[0]):
+            raise ValueError("mask, scores and labels must have the same shape!")
+
+        to_keep = labels.ne(num_labels) & (scores > object_mask_threshold)
+
+        return masks[to_keep], scores[to_keep], labels[to_keep]
+
+    def post_process_semantic_segmentation(
+        self, outputs: "MaskFormerForInstanceSegmentationOutput", target_size: Tuple[int, int] = None
+    ) -> "torch.Tensor":
+        """
+        Converts the output of [`MaskFormerForInstanceSegmentationOutput`] into semantic segmentation predictions. Only
+        supports PyTorch.
+
+        Args:
+            outputs ([`MaskFormerForInstanceSegmentationOutput`]):
+                The outputs from [`MaskFormerForInstanceSegmentation`].
+
+        Returns:
+            `torch.Tensor`: A tensor of shape `batch_size, height, width`.
+        """
+        segmentation = self.post_process_segmentation(outputs, target_size)
+        semantic_segmentation = segmentation.argmax(dim=1)
+        return semantic_segmentation
+
+    def post_process_panoptic_segmentation(
+        self,
+        outputs: "MaskFormerForInstanceSegmentationOutput",
+        object_mask_threshold: float = 0.8,
+        overlap_mask_area_threshold: float = 0.8,
+        is_thing_map: Optional[Dict[int, bool]] = None,
+    ) -> List[Dict]:
+        """
+        Converts the output of [`MaskFormerForInstanceSegmentationOutput`] into image panoptic segmentation
+        predictions. Only supports PyTorch.
+
+        Args:
+            outputs ([`MaskFormerForInstanceSegmentationOutput`]):
+                The outputs from [`MaskFormerForInstanceSegmentation`].
+            object_mask_threshold (`float`, *optional*, defaults to 0.8):
+                The object mask threshold.
+            overlap_mask_area_threshold (`float`, *optional*, defaults to 0.8):
+                The overlap mask area threshold.
+            is_thing_map (`Dict[int, bool]`, *optional*):
+                Dictionary mapping class indices to either `True` or `False`, depending on whether or not they are a
+                thing. If not set, defaults to the `is_thing_map` of ADE20K-150 panoptic.
```

Comment:
```suggestion
                thing. If not set, defaults to the `is_thing_map` of COCO panoptic.
```
In the code, it seems like one defaults to COCO panoptic.

Reply:
It is